src/giskardpy/plugin_world_visualization.py

```diff
@@ -0,0 +1,62 @@
+import hashlib
+
+import py_trees
+import rospy
+from visualization_msgs.msg import Marker, MarkerArray
+
+from plugin import GiskardBehavior
```

Comment:
if pycharm did this import for you, then your setup is probably not right.
it should be `from giskardpy.plugin import GiskardBehavior`